docs/exposing-binding-data/intro-expose-binding.md

```diff
@@ -4,4 +4,82 @@ sidebar_position: 1
 
 # Introduction
 
+Connecting applications to backing services is always a challenge as each
+service provider suggests a different way to access their secrets and consume
+them in an application.  Service Binding helps the developers by providing a
+consistent and predictable experience and removes error-prone usual manual steps
+to configure the binding between an application and backing services.
 
+For the Service Binding Operator to provide the binding metadata, the backing
+services must expose the binding metadata in a way that can be detected by
+Service Binding Operator.
+
+Binding metadata is exposed for the following scenarios:
+ 
+* **The backing service is considered as a [Provisioned
+  Service][provisioned-service]** resource: The service you intend to connect
+  to, is compliant with the Service Binding specification and the detection of
+  all the binding metadata values is automatic.
+
+* **The backing service is not a [Provisioned Service][provisioned-service]**
+  resource: You must expose the binding metadata from the backing service.  You
+  can expose the binding metadata using any of the following methods:
+  * **[Direct Secret Reference][direct-secret-reference]**: When all the
+    required binding values are available in a `Secret` that you can refenrence
+    in your Service Binding definition.
+  * **Generate Secret through CRD or CR annotations**: When you can annotate the
+    resources of the backing service, to expose the binding metadata with
+    specific annotations.
+  * **Generate Secret through OLM descriptors**: when you can provide OLM
+    descriptors, if the backing service is provided by an Operator.
+  * **Detect Binding metadata through owned resources**: when the backing
+    service owns Kubernetes resources such as `Service`, `Route`, `ConfigMap` or
+    `Secret` that you can use to detect the binding metadata.
+
+Service Binding Operator provides the ability to extract the values from the
+backing service resources and CRDs.  You can use the following methods to map
+and extract the binding metadata:
+
+- Extract a string from a resource
```

Comment:
Suggestion: 

- Replace all the instances of "extract" to "expose" (this includes their various verb forms as well, for example: extracted, extracting). Apply this change globally in the PR.

Reply:
Done. 5692973650f7aef41e870c8aa40307d969e64cb6